### DISCUSSION TITLE
How to access SSL/TLS settings in Http tool?

### DISCUSSION THREAD

**Discussion by kghenderson** [cue-lang/cue#1557] [Q&A] [answered]
Hi All,

I'm trying to issue a simple Get request using pkg/tool/http. The command is executing properly, but I'm getting an "x509: certificate is not valid for any names" error. Unfortunately I don't have access to change the certs on this server, but it's internal only service (not internet facing), and the cert is provided by the software provider. 

I need to disable that certificate check. This seems the way to do it, i.e. with TLSConfig & InsecureSkipVerify, but I'm not sure if/how that can be done with the cue package:

https://stackoverflow.com/questions/71292261/golang-x509-cannot-validate-certificate

Any help would be greatly appreciated, thank you :-)

Kevin

**Comment by eonpatapon** [comment, marked as answer]
Not supported yet, but marked as TODO (https://github.com/cue-lang/cue/blob/master/pkg/tool/http/http.go#L77)

Feel free to open an issue if you need that feature.

**Reply by kghenderson** [reply]
I see it in the todo notes there now, thanks @eonpatapon

**Reply by kghenderson** [reply]
created an issue here:  https://github.com/cue-lang/cue/issues/1558
